### PR TITLE
feat: added digi technology

### PR DESCRIPTION
**Pull request by trishan9**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added new company
- [ ] Other(s)

### CODE REVIEW

File: data/digi-technology.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Digi Technology Nepal",
+    "establishedAt": 757361700,
+    "location": "Dillibazar, Kathmandu",
+    "website": "https://digitechnologynepal.com/",
+    "email": "info@digitechnologynepal.com",
+    "phone": "+977 9801986455",
```
Comment (thenishantsapkota) [inline data/digi-technology.json:7]:
```suggestion
    "phone": "+977-9801986455",
```
Also do update the number if there is a landline number available.

File: data/digi-technology.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "socials": {
+      "facebook": "https://www.facebook.com/digitechepal",
+      "github": null,
+      "instagram": null,
+      "linkedin": "https://www.linkedin.com/company/digi-technology-nepal/",
+      "twitter": null,
+      "youtube": null
+    }
+  }
+
```
Comment (thenishantsapkota) [inline data/digi-technology.json:17]: Add a new line at the end of the file.

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]
Also please properly format the code using prettier.

**Comment by trishan9** [comment]
@thenishantsapkota  done

**Review by thenishantsapkota — APPROVED** [review]

**Comment by thenishantsapkota** [comment]
Thank you so much for your contributions @trishan9  :heart:

**Pull request merged**